test/functional/editions_controller_test.rb

```diff
@@ -264,6 +264,46 @@ class EditionsControllerTest < ActionController::TestCase
     end
   end
 
+  context "#duplicate" do
+    setup do
+      @answer = FactoryBot.create(:answer_edition, panopticon_id: FactoryBot.create(:artefact).id)
```

Comment:
I don't believe it's necessary to set the `panopticon_id` for the purposes of these tests.
```suggestion
      @answer = FactoryBot.create(:answer_edition)
```